hal/src/main/java/edu/wpi/first/hal/DriverStationJNI.java

```diff
@@ -91,6 +61,53 @@ public static void getControlWord(ControlWord controlWord) {
         ((word >> 5) & 1) != 0);
   }
 
+  /**
+   * Adds an operating mode option.
+   *
+   * @param name name of the operating mode
+   * @return unique ID used to later identify the operating mode; if an empty string is passed,
+   *         0 is returned; identical names result in identical IDs
+   */
+  public static native int addOpMode(String name);
+
+  /**
+   * Removes an operating mode option.
+   *
+   * @param name name of the operating mode
+   */
+  public static native void removeOpMode(String name);
+
+  /**
+   * Clears all operating mode options.
+   */
+  public static native void clearOpModes();
+
+  /**
+   * Gets the current operating mode of the driver station.
+   *
+   * @return the unique ID provided by the addOpMode() function, or 0 to indicate the robot is
+   *         disabled (no active operating mode)
+   */
+  public static native int getOpMode();
```

Comment:
I was calling this "CurrentOpMode" in the comm lib to make it a bit clearer. Should we do that here too?

Reply:
Or maybe "ActiveOpMode"?